packages/hub/src/scope.ts

```diff
@@ -297,6 +300,21 @@ export class Scope implements ScopeInterface {
     return this._session;
   }
 
+  /**
+   * @inheritDoc
+   */
+  public getParent(): Scope | undefined {
+    return this._parent;
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public setParent(parentScope?: Scope): this {
+    this._parent = parentScope;
+    return this;
+  }
```

Comment:
I'd prefer not to add even more API surface to `Scope`.

I can't really assess the impact of this in practice, but note this could prevent otherwise unused scopes to be garbage-collected.

Specially opposed to adding this to a generic package that affects every JS SDK, when apparently the need is only within NextJS (and I think we don't really need it).